src/ledger/AccountFrame.cpp

```diff
@@ -353,6 +356,47 @@ AccountFrame::countObjects(soci::session& sess)
     return count;
 }
 
+uint64_t
+AccountFrame::countObjects(soci::session& sess,
+                           uint32_t oldestLedger,
+                           uint32_t newestLedger)
+{
+    uint64_t count = 0;
+    sess << "SELECT COUNT(*) FROM accounts"
+            " WHERE lastmodified >= :v1 AND lastmodified <= :v2;",
+         into(count), use(oldestLedger), use(newestLedger);
+    return count;
+}
+
+void
+AccountFrame::deleteAccounts(Database& db, uint32_t oldestLedger)
```

Comment:
By the name and argument types I would expect that to remove accounts on specific ledger, not from to.
I think it needs better name, like deleteAccountModifiedOnOrAfter (seems long) or deleteAccountFrom (not as clear).

Do you have any better idea?